frontend/src/components/about/CitationWidget.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="is-flex is-justify-content-flex-start">
+    <a
+      :href="plumxref()"
+      data-popup="right"
+      data-size="medium"
+      class="plumx-plum-print-popup"
+      data-site="plum"
+      data-hide-when-empty="true"
+    >
+      An atlas of human metabolism
+    </a>
+    <div
+      data-badge-popover="right"
+      data-badge-type="donut"
+      :data-doi="doi"
+      data-hide-no-mentions="true"
+      class="altmetric-embed px-2"
+    ></div>
+    <div
+      class="scite-badge px-2"
+      :data-doi="doi"
+      data-layout="vertical"
+      data-show-zero="false"
+      data-small="true"
+    ></div>
+    <div class="__dimensions_badge_embed__ px-2" :data-doi="doi" data-style="small_circle"></div>
```

Comment:
Forgot to mention, I think it looks slightly better with the one that's most unlike others at the end. Just a personal preferences so please feel free to ignore.

```vue
    <div class="__dimensions_badge_embed__ px-2" :data-doi="doi" data-style="small_circle"></div>
    <div
      class="scite-badge px-2"
      :data-doi="doi"
      data-layout="vertical"
      data-show-zero="false"
      data-small="true"
    ></div>
```

<img width="802" alt="image" src="https://user-images.githubusercontent.com/423498/159291591-201c0343-dffb-41e5-9bd8-a44f2e9644fd.png">

Reply:
I actually prefer the current positioning, more pleasant to look at for some reason IMO. @MalinAhlberg @nanjiangshu Which version do you prefer?

Reply:
I vote for Shan's version 😄

Reply:
Haha ok, the majority has spoken then!